pytorch_lightning/trainer/trainer.py

```diff
@@ -113,7 +113,7 @@ def __init__(
             use_amp=False,  # backward compatible, todo: remove in v0.9.0
             precision: int = 32,
             print_nan_grads: bool = False,  # backward compatible, todo: remove in v0.9.0
-            weights_summary: str = 'full',
+            weights_summary: Optional[str] = 'full',
```

Comment:
what is the case of being None?

Reply:
It will turn of the summary. See [here](https://pytorch-lightning.readthedocs.io/en/0.7.1/trainer.html#weights-summary).